src/content/developers/tutorials/how-to-mint-an-nft/index.md

```diff
@@ -32,7 +36,7 @@ Inside your scripts directory, create an mint-nft.js file and add the following
     const { createAlchemyWeb3 } = require("@alch/alchemy-web3");
     const web3 = createAlchemyWeb3(API_URL);
```

Comment:
Note, though the indented text will render as code, but we can take advantage of the syntax highlighting if we wrap these code blocks in triple-back-ticks as you've done in the others, put the programming language immediately after the opening back-ticks.  (same for code blocks below)